app/templates/main/volunteerProfile.html

```diff
@@ -0,0 +1,117 @@
+{% extends "base.html"%}
+{% block scripts %}
+  {{super()}}
+  <script src="//cdnjs.cloudflare.com/ajax/libs/timepicker/1.3.5/jquery.timepicker.min.js"></script>
+  <script src="/static/js/volunteerProfile.js"></script>
+  <script type="text/javascript" src="{{url_for('static', filename='js/ .js') }}?u={{lastStaticUpdate}}"></script>

```

Comment:
This is not including anything. Where did this come from?

Reply:
Maybe from Ebeneezer's branch when we merged with it at the beginning. We trusted that what he had was necessary for his code to work and didn't think of refactoring it at the time. These scripts will be moving away from our files soon enough.

Reply:
They have been removed.